1.translate contribute section into chinese

inolddays opened the pull request:
2.modify concepts _index.md
3.translate zh/docs _index.md

Signed-off-by: JohnnyThree <whereshallyoube@gmail.com>

inolddays committed:
1.translate contribute section into chinese
2.modify concepts _index.md
3.translate zh/docs _index.md

Signed-off-by: JohnnyThree <whereshallyoube@gmail.com>

morgo commented on content/zh/docs/concepts/_index.md:
```diff
@@ -1,23 +1,6 @@
-notoc: true
+title: Concepts
+description: 学习Vitess核心的概念和术语
+aliases: ['/docs/overview/concepts/']
```
We need to remove this aliases line, otherwise there will be a URL redirecting to both the english and chinese version of the page.

inolddays replied:
sorry. forget this. will fix very soon

inolddays replied:
fixed

morgo commented on content/zh/docs/contributing/_index.md:
```diff
@@ -1,78 +1,83 @@
     </details>
 
-* Go through the [Vitess Kubernetes tutorial](../get-started/kubernetes).
+* 详细了解 [Vitess Kubernetes tutorial](../tutorials/kubernetes) 部分。
```
This URL should now be `../get-started/kubernetes`

inolddays replied:
fixed

morgo commented on content/zh/docs/contributing/code-reviews.md:
```diff
@@ -1,75 +1,70 @@
-We reference to them as `b/########` or `BUG=########` in commit messages and comments.
-External users can ignore these.
+团队正在解决的大多数Bug都是有在内部跟踪的。
+我们在提交消息和注释中将它们称为`b/########` 或者`BUG=########`。
```
This should have now been updated and removed from english docs. It was Google specific and is now not relevant.

inolddays replied:
removed

morgo reviewed (approved):
A couple of small comments, otherwise LGTM

inolddays committed:
1.rm extra en words in build-from-source.md
2.add url of jianshu about  "vitess build on mac"
3.modify "Vitess Kubernetes tutorial" URL

Signed-off-by: JohnnyThree <whereshallyoube@gmail.com>

inolddays committed:
1.rm aliases line of concepts
2.ajust weight of contribute section

Signed-off-by: JohnnyThree <whereshallyoube@gmail.com>

morgo commented:
LGTM, please mark as ready for review and I'll approve.
